C#/Libraries/SkillTools/SkillTools/DataStorage/BasicStorage.cs

```diff
@@ -0,0 +1,213 @@
+﻿using System;
```

Comment:
Should all of these code files have the license blurb?

Reply:
Adding a license file